### ISSUE TITLE
Support tasty-1.5

### ISSUE THREAD

**Issue by Bodigrim** [phile314/tasty-silver#37]
```
tasty-silver-3.3.1.1$ cabal build -c 'tasty>=1.5'
[4 of 6] Compiling Test.Tasty.Silver.Interactive

Test/Tasty/Silver/Interactive.hs:581:25: error: [GHC-83865]
    • Couldn't match type: Ap (Reader Level) TestOutput
                     with: [Ap (Reader Level) TestOutput]
      Expected: OptionSet
                -> TestName
                -> [Ap (Reader Level) TestOutput]
                -> Ap (Reader Level) TestOutput
        Actual: OptionSet
                -> TestName
                -> Ap (Reader Level) TestOutput
                -> Ap (Reader Level) TestOutput
    • In the expression: handleGroup
      In a record update at fields ‘foldSingle’, ‘foldGroup’,
      with type constructor ‘TreeFold’
      and data constructor ‘TreeFold’.
      In the first argument of ‘foldTestTree’, namely
        ‘trivialFold
           {foldSingle = handleSingleTest, foldGroup = handleGroup}’
    |
581 |           , foldGroup = handleGroup
    |                         ^^^^^^^^^^^

Test/Tasty/Silver/Interactive.hs:928:31: error: [GHC-83865]
    • Couldn't match expected type: Int -> Maximum Int
                  with actual type: [Int -> Maximum Int]
    • In the first argument of ‘(.)’, namely ‘m’
      In the expression: m . (+ indentSize)
      In the expression: \ _ _ m -> m . (+ indentSize)
    |
928 |       , foldGroup = \_ _ m -> m . (+ indentSize)
    |                               ^
Error: cabal: Failed to build tasty-silver-3.3.1.1.
```

**Comment by andreasabel** [comment]
I revised the published 3.2 and 3.3 to require `tasty < 1.5`:
* tasty-silver-3.2-r3
* tasty-silver-3.2.1-r3
* tasty-silver-3.2.2-r3
* tasty-silver-3.2.3-r2
* tasty-silver-3.3-r2
* tasty-silver-3.3.1-r2
* tasty-silver-3.3.1.1-r1

**Comment by andreasabel** [comment]
Released 3.3.1.2 which supports `tasty-1.5`.
